libs/ledger/src/storage_unit/transaction_store_sync_protocol.cpp

```diff
@@ -211,6 +205,12 @@ TSSP::TxArray TransactionStoreSyncProtocol::PullSpecificObjects(DigestSet const
     }
   }
 
+  if (!ret.empty())
```

Comment:
This should be here (printing out if there is a specific pull needed, not normal pull)